mmcv/ops/csrc/common/cuda/convex_iou_cuda_kernel.cuh

```diff
@@ -18,9 +18,37 @@ struct Point {
   double x, y;
   __device__ Point() {}
   __device__ Point(double x, double y) : x(x), y(y) {}
+  __device__ friend bool operator<(const Point& lhs, const Point& rhs) {
```

Comment:
__forceinline__ small function to reduce function call cost.

Reply:
Yeah, I add the `inline` keyword.